Dashboard/app/js/templates/application/navigation.handlebars

```diff
@@ -2,11 +2,11 @@
   {{#view view.NavItemView item="navSurveys" }}
     <a {{action doNavSurveys}}>{{t _surveys}}</a>
   {{/view}}
-	{{#if view.showDevicesButton}}
   {{#view view.NavItemView item="navDevices" }}
+	{{#if view.showDevicesButton}}
     <a {{action doNavDevices}}>{{t _devices}}</a>
-	{{/view}}
   {{/if}}
+	{{/view}}
```

Comment:
The indentation looks a bit weird here. can you check it? and the `{{if ..}}` statement before it.